src/parsley/parsley.py

```
@@ -209,6 +212,7 @@ def format_line(parsed_data: dict) -> str:
     msg_type = parsed_data['msg_type']
     board_type_id = parsed_data['board_type_id']
     board_inst_id = parsed_data['board_inst_id']
+    msg_metadata = parsed_data['msg_metadata']
```

Ruff (0.15.0)

[error] Local variable `msg_metadata` is assigned to but never used

Remove assignment to unused variable `msg_metadata`

(F841)